bin/start_tests.sh

```diff
@@ -11,7 +11,21 @@ deactivate
 # Move down to the dcoscli package
 cd cli
 
+# overwrite core.dcos_url in the dcos.toml
+cp tests/data/dcos.toml $DCOS_CONFIG
+sed -i "s/change.dcos.url/$VBOX_IP/g" $DCOS_CONFIG
```

Comment:
It gets set in the TC agent's configuration. We have a filter so that CLI builds only run on agents that have that env variable set.